libs/search-widget/src/_video-widget/VideoTile.svelte

```diff
@@ -125,31 +137,27 @@
   };
 </script>
 
-<svelte:window bind:innerWidth/>
-<div class="video-tile"
-     class:expanded
-     class:showAllResults
-     bind:this={tileElement}
-     style:max-height="{expandedHeight}"
+<svelte:window bind:innerWidth />
+<div
+  class="sw-video-tile"
+  class:expanded
+  class:showAllResults
+  bind:this={tileElement}
+  style:max-height={expandedHeight}
```

Comment:
I noticed this kind of reformatting in a lot of places, would be nice if there was a way to configure your formatter to always keep the first attribute on the same line as the tag…